schemas/user_preview.yaml

```diff
@@ -9,4 +9,6 @@ user_preview:
       type: string
     name:
       type: string
+    email:
```

Comment:
I think we have removed that property again from the previews? If not, I need to check again where this is added. I was asked that the email is only exposed through the user details, not the preview.

Reply:
@forelabs it was removed from GraphQL i think, but we expose `email` on various endpoints (e.g. user, member, everything that is using `UserSerializer`). I added it back in, since the API is still returning this data.